webextensions/api/action.json

```
@@ -571,7 +571,7 @@
               "description": "The `text` property of the `details` parameter can be set to `null`.",
               "support": {
                 "chrome": {
-                  "version_added": false
+                  "version_added": "116"
```

Based on https://chromiumdash.appspot.com/commit/09471bf4bedf979b846dc2e51f259602e317ddc7 and https://github.com/chromium/chromium/commit/09471bf4bedf979b846dc2e51f259602e317ddc7 it landed in Chrome 83.

```suggestion
                  "version_added": "83"
```

The parent feature, `setBadgeText`, therefore landed earlier than 88, and in fact it looks like it landed in Chrome 4: https://github.com/chromium/chromium/commit/5132483c2e2775f0e6bcc249e1438cf88dac97a2, i.e. [4.0.221.8](https://github.com/chromium/chromium/releases/tag/4.0.221.8) before Chrome 4.0.223.